src/corrgi/alignment.py

```diff
@@ -15,7 +20,7 @@
 ]
 
 
-def autocorrelation_alignment(catalog: Catalog) -> PixelAlignment:
+def autocorrelation_alignment(catalog: Catalog, bins=None) -> PixelAlignment:
```

Comment:
This looks great! 

From what I understood, we are calculating the pixels' boundaries and then considering only those pairs of pixels that have boundary points whose distances fall into at least one of the bins. Can we please have some comments explaining the logic in more detail?